scalableminds/webknossos
# Upgrade to Play 2.8 and sbt 1.4

Update more backend dependencies to decrease technical debt and security vulnerabilities
`play`, `akka`, `sbt`, `postgresql`, `scalatestplus-play`

The remaining security vulnerabilities marked as HIGH or CRITICAL as reported by dependency-check are caused by play-silhouette (no new release available yet) and webknossos-wrap (I’ll look into that in the coming days, I don’t think newrelic is required there anymore at all, and updating the logger can’t be hard)

Note that this PR is based on #5512 (Depending on reviews I’ll either merge this one into that or change the base as required)

### Dev Deployment

https://play.webknossos.xyz/

### Steps to test:
- CI should run through
- trace some, save, download, upload
- register new user
- try around both on deployed dev instance and locally


------
